scripts/build-prep-1node-cortx-mgw.sh

```diff
@@ -71,13 +71,13 @@ echo "========================================="
 
 echo
 echo 'update singlenode.yaml with devices in "path: /dev/loop0" format'
-echo 'Do not use "/dev/$available_device" in cluster yaml'
+echo "Do not use "/dev/$available_device" in cluster yaml"
```

Comment:
Quoting is confusing here. Do it like this:
```
echo "Do not use /dev/$available_device in cluster yaml"
```

Reply:
Fixed it.